spec/controllers/activities_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
```

Comment:
Please use single quotes whenever you don't have string interpolation. That's a style thing